apps/editor.planx.uk/src/pages/FlowEditor/lib/store/index.ts

```diff
@@ -30,7 +30,10 @@ export declare namespace Store {
     data?: Record<string, any>;
     auto?: boolean;
     override?: Record<string, any>;
+    createdAt?: string;
+    seq?: number;
   };
+  // TODO: Wrap UserData in Required<T>
```

Comment:
Intentional TODO I'll look at in future (outside the scope of this PR).

`Breadcrumbs` will always contain a `createdAt` and `seq`, but `UserData` is used more broadly across the application (it exists as a partial object, before these fields are added).

Rather than loosely type these as optional fields, it's probably better to express them as they would be interacted with within PlanX.